docs/include/conf/events/writing-day.rst

```diff
@@ -1,14 +1,7 @@
-Writing Day
------------
```

Comment:
This might break previous years, if we didn't include the headers in the main rst file.